website/docs/gitops-templates/templates.mdx

```diff
@@ -136,6 +136,74 @@ Where
   - `namespace` -  (optional) is the default target namespace for the profile
   - `editable` -  (optional, default=false), allow the user to de-select this profile, making it a default instead of a requirement.
 
+## Template paths
+
+### Specifying paths
+
+The `spec.resourcetemplates[].path` field can be used to specify the paths of the rendered template resources.
+This allows more control over where different resources in the template are rendered.
+
+- The path is relative to the repository root.
+- The path can be templated using params
+
+Example:
+```yaml
+spec:
+  resourcetemplates:
+    // highlight-next-line
+    - path: ./clusters/${CLUSTER_NAME}/definition/cluster.yaml
+      content:
+        - apiVersion: cluster.x-k8s.io/v1alpha4
+          kind: Cluster
+          metadata:
+            name: ${CLUSTER_NAME}
+          ...
+        - apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+          kind: AWSCluster
+          metadata:
+            name: ${CLUSTER_NAME}
+          ...
+    // highlight-next-line
+    - path: ./clusters/${CLUSTER_NAME}/workloads/helmreleases.yaml
+      content:
+        - apiVersion: helm.toolkit.fluxcd.io/v2beta1
+          kind: HelmRelease
+          metadata:
+            name: ${CLUSTER_NAME}-nginx
+          ...
+        - apiVersion: helm.toolkit.fluxcd.io/v2beta1
+          kind: HelmRelease
+          metadata:
+            name: ${CLUSTER_NAME}-cert-manager
+          ...
+```
+
+### Default paths
+
+If the `spec.resourcetemplates[].path` is omitted a default path for the rendered template is calculated.
+In this case some some of the submitted params are used. You must provide one of the following parameters:
+- `CLUSTER_NAME`
```

Comment:
Only are required of you are not specifying the path correct?

Reply:
Yes, _or_ if you are using profiles/kustomizations.